x-pack/plugins/security_solution/public/management/components/endpoint_responder/components/offline_callout.tsx

```diff
@@ -40,7 +64,7 @@ export const OfflineCallout = memo<OfflineCalloutProps>(({ endpointId }) => {
             <FormattedMessage
               id="xpack.securitySolution.responder.hostOffline.callout.body"
               defaultMessage="The host {name} is offline, so its responses may be delayed. Pending commands will execute when the host reconnects."
-              values={{ name: <strong>{endpointDetails.metadata.host.name}</strong> }}
+              values={{ name: <strong>{hostName}</strong> }}
```

Comment:
Note that I don't know if the statement here is actually accurate - meaning: will SentinelOne actually sent the `isolate`/`release` to the host once it comes back online.  Can you test that?

Reply:
Good point. I tested it out by:

1. Stopping the multipass VM.
2. Waited to see the offline callout.
3. Sent an isolate request (it says action completed on the console output, but action status badge says isolating)
4. Restarted the VM after it had stopped completely.
5. I did a ping request on the VM.
6. The agent came online on the responder and the callout disappeared, and then the agent got isolated while pings were in flight.
7. The status showed that the agent was isolated. 
8. Also the shell existed out of the VM instance, so it seems to work as expected.
9. Trying to shell into the agent again didn't work as it is isolated. This was expected.

From here, I stopped the agent and tried the release action and followed the same steps as above. It worked as expected.

Reply:
awesome. thank you for validating that 🎉